cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -92,6 +92,26 @@ spec:
     # To signify that automatic deletion is desired, use the value "yes-really-destroy-data". Only this and an empty
     # string are valid values for this field.
     deleteDataDirOnHosts: ""
+
+    # sanitizeDisk represents whether we should sanitize OSD disks on cluster deletion
+    sanitizeDisk:
```

Comment:
What about this schema? 
- There is a single "yes-really-destroy-data" value that enables all the cleanup policy
- The utility dd or shred isn't exposed in the schema. 
- Settings that only apply to shred are in a subsection
- No need for the `sanitize` prefix on all the settings
- `sanitizeDisks` is plural
```yaml
cleanupPolicy:
  enabled: "yes-really-destroy-data"
  deleteDataDirOnHosts: true
  sanitizeDisks:
    enabled: true | false
    method: metadata | full
    utility: zero | random
    randomConfig: # uses shred
      source: random | urandom
      iterations: 1
```

Reply:
I'm also concerned about backporting this full feature to 1.3. This feature is really better to delay for 1.4 so we have more time for feedback so we don't have to worry about backward compatibility on something that didn't have time for design review. How about breaking this into two PRs where only one would be backported? For 1.3 I'd suggest we backport only this schema and only dd with zero'ing would be supported to start with. 
```yaml
cleanupPolicy:
  enabled: "yes-really-destroy-data"
  deleteDataDirOnHosts: true
  sanitizeDisks:
    enabled: true | false
    method: metadata | full
```

Reply:
> What about this schema?
> 
> * There is a single "yes-really-destroy-data" value that enables all the cleanup policy
> * The utility dd or shred isn't exposed in the schema.
> * Settings that only apply to shred are in a subsection
> * No need for the `sanitize` prefix on all the settings
> * `sanitizeDisks` is plural
> 
> ```yaml
> cleanupPolicy:
>   enabled: "yes-really-destroy-data"
>   deleteDataDirOnHosts: true
>   sanitizeDisks:
>     enabled: true | false
>     method: metadata | full
>     utility: zero | random
>     randomConfig: # uses shred
>       source: random | urandom
>       iterations: 1
> ```

Actually the `source` can be set to `random` with `dd` as well so it's not accurate. It'd just keep how it is today and not add `randomConfig`. Although I can simplify the naming.

Reply:
Per our discussion, the suggested schema could be:
```yaml
cleanupPolicy:
  confirmation: "yes-really-destroy-data"
  sanitizeDisks:
    method: metadata | full
    utility: zero | random
    randomConfig: # uses shred
      source: random | urandom
      iterations: 1
```
- Name it "confirmation" instead of "enabled". This is the only change to the cluster CR that would be backported.
- No need for `deleteDataDirOnHosts` or `sanitizeDisks.enabled`. If they really want us to destroy data, we will just destroy everything related to Rook. No need for separate policies.
- All policy settings for sanitizeDisks would remain in master and not be backported.